### DISCUSSION TITLE
How do you track  contract verification progress

### DISCUSSION THREAD

**Discussion by wighawag** [blockscout/blockscout#9373] [Support]
when calling `/api/v2/smart-contracts/${address}/verification/via/standard-input`

I get the response:

```
{ message: 'Smart-contract verification started' }
```

How do I track progress ?

**Comment by fedor-ivn** [comment]
AFAIK, the frontend establishes the separate web socket. The verification confirmation response goes over that one.

**Comment by wighawag** [comment]
is this documented somewhere ?
